### ISSUE TITLE
Make Recently viewed section of the new search modal scrollable

### ISSUE THREAD

**Issue by frakic** [vector-im/element-web#22551]
#### What would you like to do?
If there's more Recently viewed items than can fit on screen, they can be accessed by pressing keyboard arrows but there's no way to access them with a mouse.
#### Why would you like to do it?
I think the modal should be accessible with both a keyboard and a mouse.
#### How would you like to achieve it?
Maybe add a horizontal scroll bar?

### Have you considered any alternatives?

If we don't want a scroll bar, we could implement some clickable arrows, which would allow the user to navigate the Recently viewed section.

### Additional context

This can only be observed if you have more than 9 recently viewed items or you resize your window.

**Comment by HarHarLinks** [comment]
I agree it should be scrollable by using the mouse, which does not appear to be possible at all right now.
There are multiple options, including nested scrollbars. I think my favorite idea would be limiting the current scrollbar to the recent searches and add a horizontal one to the recently viewed. Once you start typing, you should get the current behavior.

> If we don't want a scroll bar, we could implement some clickable arrows

I think the ability to scroll beyond the items shown on screen was introduced based on my enhancement request, as was the horizontal navigation by arrow keys. However a visual indicator of the ability to scroll beyond the shown items has not yet been considered. This would be implied by adding a scroll bar. Alternatively I remember making a mockup I recreated here:
![image](https://user-images.githubusercontent.com/2803622/173583067-c65dbdaf-9792-41d6-8329-ddbfac14d687.png)
Arrow indicators pop up when there is something to be scrolled, e.g. initially we're all the way to the left so there does not need to be an arrow on the left.
